build.sbt

```diff
@@ -56,7 +52,7 @@ lazy val microservice = Project(appName, file("."))
     IntegrationTest / testGrouping := oneForkedJvmPerTest((IntegrationTest / definedTests).value),
     IntegrationTest / parallelExecution := false
   )
-  .settings(scalaVersion := "2.12.12")
+  .settings(scalaVersion := "2.13.8")
```

Comment:
Can we go straight to 2.13.10? If the platform has specified 2.13.8 then no worries.

Reply:
The guide mentioned 2.13.8 & the catalogue also did so I just went for that. I can have have a look if 2.13.10 is compatible with our dependencies.

Reply:
Ah, I see it in the guide screenshot. 2.13.10 came out after they published the guide and 2.13.9 was only two days before.

I think it's probably best to be on the newest available version, but have no issues with 2.13.8 if our libraries aren't compatible or something.